bloom/generators/debian/generator.py

```diff
@@ -207,6 +207,26 @@ def format_depends(depends, resolved_deps):
     return formatted
 
 
+def format_description(value):
+    """
+    Format proper <synopsis, long desc> string following Debian control file
+    formatting rules. Treat first line in given string as synopsis, everything
+    else as a single, large paragraph.
+
+    Future extensions of this function could convert embedded newlines and / or
+    html into paragraphs in the Description field.
+
+    https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description
+    """
+    value = debianize_string(value)
+    parts = value.split('.', 1)
```

Comment:
Yes, true.

Splitting text correctly on sentence boundaries is hard to get robust though, so I opted for this naive way of doing it with just `split(..)`. It will only work correctly for properly formatted package descriptions then though.